skimage/transform/_geometric.py

```diff
@@ -293,23 +282,10 @@ def __call__(self, coords):
         coords_homogeneous = np.column_stack([coords, np.ones(coords.shape[0])])
         return coords_homogeneous @ self.params.T
 
-    def inverse(self, coords):
-        """Apply inverse transformation.
-
-        Parameters
-        ----------
-        coords : (N, 2) array_like
-            Destination coordinates.
-
-        Returns
-        -------
-        coords : (N, 3) array
-            Epipolar lines in the source image.
-
-        """
-        coords = np.asarray(coords)
-        coords_homogeneous = np.column_stack([coords, np.ones(coords.shape[0])])
-        return coords_homogeneous @ self.params
+    @property
+    def inverse(self):
+        """Return a transform object representing the inverse."""
+        return type(self)(matrix=self.params.T)
```

Comment:
Why is the inverse given by the transpose?

Reply:
See the discussion above: https://github.com/scikit-image/scikit-image/pull/6926#discussion_r1190949714

`__call__` does `return coords_homogeneous @ self.params.T` and `inverse` did `return coords_homogeneous @ self.params`. So the new inverse transform instance should swap both.